<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maintain_product</class>
 <widget class="QWidget" name="maintain_product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>988</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>530</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Name: </string>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>650</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Price: </string>
   </property>
  </widget>
  <widget class="QLabel" name="volume_per_tick">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>890</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Volume Per Tick: </string>
   </property>
  </widget>
  <widget class="QLabel" name="target_volume">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>770</y>
     <width>721</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Target Volume: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="image">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nameButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="priceButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>620</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Price</string>
   </property>
  </widget>
  <widget class="QPushButton" name="target_volumeButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>740</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Volume</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vol_per_tickButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>860</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume per Tick</string>
   </property>
  </widget>
  <widget class="QLabel" name="PumpStatus">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pump Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pump_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="VolumeDispensed">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Volume Dispensed: </string>
   </property>
  </widget>
  <widget class="QLabel" name="vol_dispensed_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
